### DISCUSSION TITLE
[GLPI 10.0.10] Message : Attention! Pas de widgets trouvés, merci d'en ajouter

### DISCUSSION THREAD

**Discussion by babydunet** [InfotelGLPI/mydashboard#152] [Q&A]
Bonjour,

Lorsque je veux ouvrir My Dshbord j'ai ce message qui s'affiche: "Attention! Pas de widgets trouvés, merci d'en ajouter" comment résoudre ce problème svp ?

Merci à tous

**Comment by jalcalam** [comment]
I have the same problem, uninstalling plugin, checking that there are no tables in glpi of reference and reinstalling it again the same message still comes out. GLPI version 10.0.10

**Comment by Ledzep29** [comment]
Bonjour, 

J'ai le même souci avec la une version 11.0.5 de GLPI. Une solution ? 
merci
